modules/repository/repo.go

```diff
@@ -101,6 +107,11 @@ func MigrateRepositoryGitData(ctx context.Context, u *user_model.User,
 				}
 			}
 		}
+		if git.CheckGitVersionAtLeast("2.18") == nil {
```

Comment:
this write commit graph code exists multible times - we should move it into a own func

Reply:
beside this it has my ok :)